tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -50,42 +50,30 @@ If there's no default value, you can add a type to the variable.
 
     @export var number: int
 
-Export works with resource types.
-
-::
-
-    @export var character_face: Texture
-    @export var scene_file: PackedScene
-
-There are many resource types that can be used this way, try e.g.
-the following to list them:
+Resources and nodes can be exported.
 
 ::
 
     @export var resource: Resource
+    @export var node: Node
 
-Integers and strings hint enumerated values.
-
-::
-
-    # Editor will enumerate as 0, 1 and 2.
-    @export_enum("Warrior", "Magician", "Thief") var character_class
+Grouping Exports
+----------------
 
-If type is String, editor will enumerate with string names.
+It is possible to group your exported variables inside the Inspector with the :ref:`@export_group <class_@GDScript_annotation_@export_group>` annotation.
+Groups cannot be nested, use :ref:`@export_subgroup <class_@GDScript_annotation_@export_subgroup>` to add subgroups within groups.
```

Comment:
These lines should be wrapped to keep the style of the document consistent. We don't do it everywhere, but in this particular article we do, so the formatting style should be preserved.

```suggestion
It is possible to group your exported properties inside the Inspector
with the :ref:`@export_group <class_@GDScript_annotation_@export_group>`
annotation. Every exported property after this annotation will be added to
the group. Start a new group or use ``@export_group("")`` to break out.

::

    @export_group("My Properties")
    @export var number = 3
    @export var string = ""

Second argument of the annotation can be used to only group properties
with the specified prefix.

Groups cannot be nested, use :ref:`@export_subgroup <class_@GDScript_annotation_@export_subgroup>`
to create subgroups within a group.
```